go/libkb/merkle_client.go

```diff
@@ -1090,25 +1124,70 @@ func parseMerkleUserLeaf(ctx context.Context, jw *jsonw.Wrapper, g *GlobalContex
 	return
 }
 
+func parseMerkleTeamLeaf(ctx context.Context, jw *jsonw.Wrapper, g *GlobalContext) (leaf *MerkleTeamLeaf, err error) {
+	g.Log.CDebugf(ctx, "+ ParsingMerkleUserLeaf")
+
+	if jw == nil {
+		g.Log.CDebugf(ctx, "| empty leaf found; team wasn't in tree")
+		leaf = &MerkleTeamLeaf{}
+		return
+	}
+
+	l, err := jw.Len()
+	if err != nil {
+		return
+	}
+	if l != 4 {
+		err = fmt.Errorf("Expected an array of length 4 but got %v", l)
+		return
+	}
```

Comment:
Is this something we necessarily want to be strict about? We might need to add more fields in the future. I see we're strict with `parseTriple` too, so I guess my question is really about that one. What do you think, @maxtaco?